src/app/auth/shared/auth.service.ts

```diff
@@ -67,4 +71,17 @@ export class AuthService {
   isTherapistLogin() {
     return this.fireAuth.auth.currentUser.email === AuthService.THERAPIST_EMAIL;
   }
+
+  /**
+   * Deletes the selected user from the DB.
+   * @param {UserModel} user
+   */
+  deleteUser(user: UserModel) {
+    secondaryApp.auth().signInAndRetrieveDataWithEmailAndPassword(user.email, AuthService.USER_PASSWORD)
```

Comment:
We need to speak about this at some point, I really dislike the idea of hardcoding the user password in order to first login and then to delete the user.
Perhaps we can use cloud functions?